pkg/api/kubegraph/analysis/pod.go

```diff
@@ -27,7 +36,47 @@ func FindRestartingPods(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 		}
 
 		for _, containerStatus := range pod.Status.ContainerStatuses {
-			if containerStatus.RestartCount >= RestartThreshold {
+			switch {
+			case containerCrashLoopBackOff(containerStatus):
+				var suggestion string
+				switch {
+				case containerIsNonRoot(pod, containerStatus.Name):
+					suggestion = D(`
+						The container is starting and exiting repeatedly, which usually means the container cannot
+						start, is misconfigured, or is unable to perform an action due to security restrictions on
+						the container. The container logs may contain messages indicating the reason the pod cannot
+						start.
+
+						This container is being run as as a non-root user due to administrative policy, and
+						some images may fail expecting to be able to change ownership or set permissions on
+						directories. Your administrator may need to grant permission for you to run root
+						containers.`)
+				default:
+					suggestion = D(`
+						The container is starting and exiting repeatedly, which usually means the container cannot
+						start, is misconfigured, or is unable to perform an action due to security restrictions on
+						the container. The container logs may contain messages indicating the reason the pod cannot
+						start.`)
```

Comment:
nice! I think it would also be cool to explain how to get the container logs. One problem every new user gets is that he can't get the container's logs because the container is not running, and at first it is not easy to understand that we need to use the "-p" option to get the logs from the previous container.

perhaps we can add something like:
> To see the container logs, use the following command: `oc logs pod -c container -p`